index.html

```diff
@@ -28,9 +28,9 @@
     <link href="js/vendor/touch-spinner/jquery.bootstrap-touchspin.min.css" rel="stylesheet" type="text/css" />
     <!-- HTML5 shim and Respond.js IE8 support of HTML5 elements and media queries -->
     <!--[if lt IE 9]>
+    <![endif]-->
```

Comment:
@CTDisasterManagement I think we need this where it was. We don't want to load respond.min.js or html5shiv unless IE needs it.